predicators/envs/pybullet_blocks.py

```diff
@@ -258,6 +258,15 @@ def _reset_state(self, state: State) -> None:
                 block_id, [bx, by, bz],
                 self._default_orn,
                 physicsClientId=self._physics_client_id)
+            # Update the block color.
+            r = state.get(block_obj, "color_r")
+            g = state.get(block_obj, "color_g")
+            b = state.get(block_obj, "color_b")
+            color = (r, g, b, 1.0)  # alpha = 1.0
```

Comment:
Default color is black? Should we just randomly sample colors if not specified them? 

Maybe we should enforce either none or all of rgb is specified?

Reply:
Oops I was on my phone and not looking clearly and it looked like `state` was some dict.

Looks in structs and it makes more sense now.